### PR TITLE
Updates to verbs dictionary and GSR

### PR DESCRIPTION
**Pull request by saumyadixit**

### CODE REVIEW

**Comment by PTB-OEDA** [comment]
Samuya,

Can you summarize these revised changes to to code in your commit?

Thx. PTB


On Apr 14, 2017 8:01 PM, "Saumya Dixit" <notifications@github.com> wrote:

------------------------------
You can view, comment on, or merge this pull request online at:

  https://github.com/openeventdata/UniversalPetrarch/pull/3
Commit Summary

   - Updates to Dictionary and GSR
   - Updates to verbs dictionary

File Changes

   - *M* Test Suite/PETR.UnitTest.records.parsing_parsed_udpipe_2.0.xml
   <https://github.com/openeventdata/UniversalPetrarch/pull/3/files#diff-0>
   (20)
   - *M* Test Suite/Test_Result.xlsm
   <https://github.com/openeventdata/UniversalPetrarch/pull/3/files#diff-1>
   (0)
   - *M* Test Suite/test_script_ud.py
   <https://github.com/openeventdata/UniversalPetrarch/pull/3/files#diff-2>
   (24)
   - *M* UniversalPetrarch/data/dictionaries/PETR.Validate.verbs.2.0.txt
   <https://github.com/openeventdata/UniversalPetrarch/pull/3/files#diff-3>
   (31)

Patch Links:

   - https://github.com/openeventdata/UniversalPetrarch/pull/3.patch
   - https://github.com/openeventdata/UniversalPetrarch/pull/3.diff

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/openeventdata/UniversalPetrarch/pull/3>, or mute the
thread

**Comment by saumyadixit** [comment]
The updates are as below : 
-- In the Test Result we have added 2 new columns - "Full Score" and "Reason". Setting "Full Score" to yes will give a score of 3/3 irrespective of the output from PETRARCH for that scenario. This is added to handle some scenarios which are there specifically to test if the code fails or not. The reason column states the reason why its set to "Full Score".
-- Update of patterns and verb codes to PETR.Validate.verbs.2.0.txt 
-- Gold standard record updated according to CAMEO2.0
